t/admin/ssl4.t

```diff
@@ -364,3 +364,40 @@ location /t {
         t.test('/apisix/admin/ssls/1', ngx.HTTP_DELETE)
     }
 }
+
+
+=== TEST 11: set ssl(sni: www.test.com) with long label
+--- yaml_config
+apisix:
+    node_listen: 1984
+    ssl:
+        key_encrypt_salt: null
+--- config
+location /t {
+    content_by_lua_block {
+        local core = require("apisix.core")
+        local t = require("lib.test_admin")
+
+        local ssl_cert = t.read_file("t/certs/apisix.crt")
+        local ssl_key =  t.read_file("t/certs/apisix.key")
+        local data = {cert = ssl_cert, key = ssl_key, sni = "www.test.com",
+         labels = {"secret-name": "js-design-test-bigdata-data-app-service-router-my-secret-number-123456"}}
```

Comment:
the parameters of the second line of functions are to the right of the left parenthesis of the first line.
```
return limit_conn_new("plugin-limit-conn", conf.conn, conf.burst,
                      conf.default_conn_delay)

```